apps/block_scout_web/lib/block_scout_web/templates/chain/show.html.eex

```diff
@@ -165,6 +81,29 @@
               <%= BlockScoutWeb.Cldr.Number.to_string!(@address_count, format: "#,###") %>
             </span>
           </div>
+
+            <%= if Map.has_key?(@chart_config, :market) do %>
+              <%= if Enum.member?(@chart_config.market, :price) do %>
+                <div class="dashboard-banner-network-stats-item dashboard-banner-network-stats-item-5">
+                    <h2 class="dashboard-banner-network-stats-label">
+                        <%= gettext "Price" %>
+                    </h2>
+                    <span class="dashboard-banner-network-stats-value" data-selector="exchange-rate" data-wei-value="<%= Explorer.Chain.Wei.from(Decimal.new(1), :ether).value %>" data-usd-exchange-rate="<%= @exchange_rate.usd_value %>">
```

Comment:
For this long line, I think we can split it to multiple lines to improve readability.

For example: 
```
<span 
    class="dashboard-banner-network-stats-value" 
    data-selector="exchange-rate" 
    data-wei-value="<%= Explorer.Chain.Wei.from(Decimal.new(1), :ether).value %>" 
    data-usd-exchange-rate="<%= @exchange_rate.usd_value %>"
></span>
```

The below line 102 has a similar issue.

Reply:
Yes.